### PR TITLE
peer_storage: tiny clean up.

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: src/raftstore/store/peer_storage.rs [hunk 1]
```diff
@@ -110,11 +110,10 @@ impl PeerStorage {
...
     pub fn initial_state(&mut self) -> raft::Result<RaftState> {
         let initialized = self.is_initialized();
-        let res: Option<HardState> = try!(self.engine
-                                              .get_msg(&keys::raft_hard_state_key(
-                                                self.get_region_id())));
+        let hs = try!(self.engine
+                          .get_msg(&keys::raft_hard_state_key(self.get_region_id())));
 
-        let (mut hard_state, found) = res.map_or((HardState::new(), false), |e| (e, true));
+        let (mut hard_state, found) = hs.map_or((HardState::new(), false), |e| (e, true));
```
Comment (BusyJay) [inline src/raftstore/store/peer_storage.rs]: s/e/s/

**Comment by ngaut** [comment]
PTAL @BusyJay

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**